### PR TITLE
update: remove hover for text element

### LINKED ISSUE #2013: Remove hover state from wallet connect request
The two components “View wallet balances…” and “Request approval…” both have a hover state, but shouldn’t (they’re just text, not interactive).

This hover state may also be on other wallet connect/tx approval screens.

image

### PR DESCRIPTION
**Pull request by jim4067**
resolves #2013

### CODE REVIEW

File: packages/app-extension/src/components/Unlocked/Approvals/ApproveOrigin.tsx [hunk 1]
```diff
@@ -48,6 +48,9 @@ const useStyles = styles((theme) => ({
     padding: "8px",
     marginBottom: "1px",
     border: `${theme.custom.colors.borderFull}`,
+    "&:hover": {
+      background: "#fff !important",
```
Comment (armaniferrante) [inline packages/app-extension/src/components/Unlocked/Approvals/ApproveOrigin.tsx:52]: I think this will not do what we want in dark mode?

Comment (aarongillett) [reply packages/app-extension/src/components/Unlocked/Approvals/ApproveOrigin.tsx:52]: I think you’re right. We want to remove the hover state, and this looks like it hard-codes one instead.

Comment (jim4067) [reply packages/app-extension/src/components/Unlocked/Approvals/ApproveOrigin.tsx:52]: thanks. hadn't thought of that.

@aarongillett that's not possible for material design elements.

File: packages/themes/src/index.tsx [hunk 2]
```diff
@@ -114,6 +114,7 @@ type CustomColors = {
   coralGradient: string;
   miniDrawerBackdrop: string;
   unreadBackground: string;
+  hoverBackground: string;
```
Comment (armaniferrante) [inline packages/themes/src/index.tsx:117]: Can we do this without introducing another field here?

Comment (jim4067) [reply packages/themes/src/index.tsx:117]: I don't think so, 

The other option I thought of would be to explicitly introduce the `palette: {mode: 'light' | 'dark' }` field in the light and dark theme definitions and using this conditionally determine the colour of the `li` element when hovered.

```ts

export const darkTheme: Partial<Theme> & { custom: { colors: CustomColors } } =
  {
    ...baseTheme,
    components: darkComponentOverrides,
    palette: {
      mode: 'dark'
    },

...

export const lightTheme: Partial<Theme> & { custom: { colors: CustomColors } } =
  {
    ...baseTheme,
    components: lightComponentOverrides,
    palette : {
      mode: "light"
    },

//do conditional rendering of which color to display on hover
marginBottom: "1px",
    border: `${theme.custom.colors.borderFull}`,
    "&:hover": {
      background: `${theme.palette?.mode === "light" ? theme.custom.colors.brandColor: theme.custom.colors.banner}`,
      opacity: "1",
    },

**Pull request closed without merging**